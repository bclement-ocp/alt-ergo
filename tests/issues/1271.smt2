; Test printing of negative real values
(set-logic ALL)
(set-option :produce-models true)
(declare-fun x () Real)
(assert (= x (- 42.0)))
(check-sat)
(get-model)
