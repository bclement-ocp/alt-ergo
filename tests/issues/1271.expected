
unknown
(
  (define-fun x () Real (- 42.0))
)
